submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/mapping/to/ToExtSubmissionMapper.kt

```diff
@@ -31,11 +35,14 @@ class ToExtSubmissionMapper(private val toExtSectionMapper: ToExtSectionMapper)
             attributes = sub.attributes.toExtAttributes(),
             collections = sub.collections.map { ExtCollection(it.accNo) },
             tags = sub.tags.map { ExtTag(it.name, it.value) },
-            stats = sub.stats.map { it.toExtStat() },
+            stats = getStats(sub.accNo),
             pageTabFiles = sub.pageTabFiles.map { it.toExtFile() },
             storageMode = sub.storageMode
         )
 
+    private fun getStats(accNo: String) =
```

Comment:
is anyone using stats section of extended? I think we should just remove it? maybe awais?

Reply:
yes, he will use it once we rely entirely on partial updates rather than exporting the submissions everyday